### PR TITLE
Migrated broadcast message page to b3

### PR DESCRIPTION
**Pull request by kkrampa**
@biyeun

### CODE REVIEW

**Comment by biyeun** [comment]
@gcapalbo this looks good to me. Since you're most familiar with this part of the codebase, can you take a quick pass and see if there'd be any issues requiring a more formal QA? I'm running it locally and it looks good

File: corehq/apps/reminders/templates/reminders/broadcasts_list.html [hunk 1]
```diff
@@ -28,37 +24,36 @@
...
+                            return '<a href="' + full[4] + '">' + full[0] + '</a>';
                         }
                     },
                     {
-                        "aTargets": [3],
-                        "fnRender": function(obj) {
-                            return '<button class="btn btn-danger" onClick="delete_broadcast(\'' + obj.aData[3] + '\');"> \
+                        "targets": [3],
+                        "render": function(data) {
+                            return '<button class="btn btn-danger" onClick="delete_broadcast(\'' + data[3] + '\');"> \
```
Comment (gcapalbo) [inline corehq/apps/reminders/templates/reminders/broadcasts_list.html]: this should just reference `data`, not `data[3]`

**Comment by gcapalbo** [comment]
I tested this locally and everything looked good and worked fine except for the following things:

- the "Previously Sent" heading should come after the first table's page navigation instead of before it
- the inputs on the broadcast create/edit page are very long, can we shorten them a bit?
- deleting a broadcast doesn't work - see my comment about using `data` instead of `data[3]` which will fix it
- the timepicker doesn't work and raises an error in the console

**Comment by gcapalbo** [comment]
(i just deleted my comment about the timepicker, i realized it wasn't working because i hadn't run bower install)

**Comment by kkrampa** [comment]
@biyeun @gcapalbo Updated

File: corehq/apps/style/crispy.py [hunk 2]
```diff
@@ -77,20 +77,43 @@ class B3MultiField(LayoutObject):
...
+        self.css_id = kwargs.pop('css_id', '')
         self.flat_attrs = flatatt(kwargs)
 
     def render(self, form, form_style, context, template_pack=None):
         template_pack = template_pack or get_template_pack()
         html = u''
+
+        errors = self._get_errors(form, self.fields)
+        if len(errors) > 0:
+            self.css_class += "has-error"
```
Comment (gcapalbo) [inline corehq/apps/style/crispy.py]: @biyeun @kkrampa should there be space around this?

**Comment by gcapalbo** [comment]
@kkrampa left just one comment, otherwise this is good from my side. will let @biyeun sign off

**Comment by kkrampa** [comment]
@biyeun @gcapalbo Fixed

**Comment by gcapalbo** [comment]
@kkrampa ok, this is good from my side

**Comment by biyeun** [comment]
@kkrampa can you resolve conflicts? otherwise, looks good to me!

**Comment by biyeun** [comment]
@kkrampa please resolve merge conflicts

**Comment by kkrampa** [comment]
@biyeun Updated

**Comment by biyeun** [comment]
:+1:

**Pull request merged**